tests/utils.go

```diff
@@ -33,6 +34,9 @@ func RunOnNode(node string, command string) (string, error) {
 	}
 
 	out, err := exec.Command("docker", "exec", provider+"-"+node, "ssh.sh", command).CombinedOutput()
+	if err != nil {
+		panic(fmt.Sprintf("failed to run docker exec command %s %s %s %s %s","docker", "exec", provider+"-"+node, "ssh.sh", command))
```

Comment:
Just out of curiosity, is there any particular reason on why not calling `fmt.Fatalf` ?

Reply:
nope it's just a POC I need to find the issue

Reply:
btw I can't find the `fmt.Fatalf` function